static/js/src/advantage/react/components/Subscriptions/SubscriptionList/RenewalSettings/RenewalSettings.tsx

```diff
@@ -15,7 +16,14 @@ const RenewalSettings = ({ positionNodeRef }: Props): JSX.Element => {
       constrainPanelWidth
       dropdownClassName="p-subscription__renewal-dropdown"
       hasToggleIcon
-      onToggleMenu={(isOpen) => setMenuOpen(isOpen)}
+      onToggleMenu={(isOpen) => {
+        setMenuOpen(isOpen);
+        sendAnalyticsEvent({
+          eventCategory: "Advantage",
+          eventAction: "subscription-auto-renewal-form",
```

Comment:
Did you think about saving these as constants (and/or adding them as types to sendAnalyticsEvent)?

Reply:
Thanks, I did think about it, but I wasn't confident enough that it would be enforcing the right names. I looked around ubuntu.com and there are a lot of different patterns for naming these actions. I think there probably needs to be an audit of the events and at that point we could probably enforce some more consistent names.